src/components/Menu/MenuLinks.tsx

```diff
@@ -33,10 +37,11 @@ export default function MenuLinks({ isSignedIn, openLogin }: IMenuLinks) {
             }}
             className="mx-2 rounded rounded-5"
           >
-            Sign up
+            {t('navigation.signUp')}
           </Button>
         </>
       )}
+      <LocaleSwitcher />
```

Comment:
Not sure if this switcher will stay here, but wrapping it in a Nav.Link will normalize it's positioning and padding with the other menu links:
```suggestion
      <Nav.Link>
        <LocaleSwitcher />
      </Nav.Link>
```